plugins/signalmonitor/signalhistorymodel.h

```diff
@@ -64,6 +65,7 @@ class SignalHistoryModel : public QAbstractTableModel
 
 public:
     enum ColumnId {
+        MonitoredColumn = 0,
```

Comment:
The extra column costs 70px here, I think that's too much. Also, "M." is not exactly an easily understandable title, even I have no idea what "M." stands for :) So, if you insist on keeping the column, can we at least shrink it to the point it hardly needs more space than the checkbox it contains?